interface/forms/newpatient/save.php

```diff
@@ -105,6 +105,22 @@
         $encounter_type_description = $option['title'];
     }
 }
+//RM - class_code can't be empty - use default class or if default not set take first in the list
+if (empty($class_code)) {
+    // use defualt from Value Set ActEncounterCode list
+    $listService = new ListService();
+    $option = $listService->getOptionsByListName('_ActEncounterCode'); //get all classes
+    // find the default
+    foreach ($option as $code) {
+        if ($code['is_default']) {
+            $class_code = $code['option_id'];
```

Comment:
```suggestion
            $class_code = $code['option_id'];
            break;
```

Reply:
thanks. yes, a break is correct here - don't know why i didn't add it in!! - will do and resubmit